srslog.go

```diff
@@ -59,6 +68,11 @@ func DialWithTLSCert(network, raddr string, priority Priority, tag string, serve
 // DialWithTLSConfig establishes a secure connection to a log daemon by connecting to
 // address raddr on the specified network. It uses tlsConfig to configure the secure connection.
 func DialWithTLSConfig(network, raddr string, priority Priority, tag string, tlsConfig *tls.Config) (*Writer, error) {
+	return dialAllParameters(network, raddr, priority, tag, tlsConfig, nil)
```

Comment:
Do we care that it's impossible to have a custom dialer _and_ a TLS config? We probably don't care, right?

Reply:
Probably not. I simply moved the implementation of DialWithTLSConfig into dialAllParameters since I noticed the others were using it, and that if they did provide a custom dialer, in my implementation, the tlsconfig wouldn't be used anyways. Sorta how if passing nil to DialWithTLSConfig from the other functions sorta ignored the config by happenstance as well.